iphone/Classes/TiUITableView.m

```diff
@@ -1602,6 +1602,18 @@ - (void)searchBarSearchButtonClicked:(UISearchBar *)searchBar
   [self makeRootViewFirstResponder];
 }
 
+- (void)setSearchText_:(id)args
+{
+  id searchView = [self.proxy valueForKey:@"searchView"];
```

Comment:
This should be a `TiViewProxy` type, so no `id` necessary.

Reply:
@m1ga copied it from `TiUIListView`: https://github.com/tidev/titanium-sdk/blob/25bb40cbb4df528303d265e94502df15178458d0/iphone/Classes/TiUIListView.m#L925

Reply:
correct. It will also not build when I remove the `id`

Reply:
why doesn't it? TiViewProxy *searchView should do it.

Regarding other usages: Just because something legacy is there, it's not best practice 😙

Reply:
Ah ok `TiViewProxy *searchView`. I've only removed the `id` part :)

That works and is changed